server/api/user/user.spec.js

```diff
@@ -19,3 +18,23 @@ describe('GET /api/users/past', function() {
       });
   });
 });
+
+
+describe('POST /api/users/', function() {
+    it('should respond with JSON array', function(done) {
+	//Create a user object without any of the required fields
+	var postData = {
+	    "user":{}
+	};
+
+	request(app)
+	    .post('/api/users/')
+	    .send(postData)
+	    .expect(422)
+	    .end(function(err, res) {
+		if(err) return done(err);
+		//res.body.should.be.instanceof(Array);
```

Comment:
let's change this to a todo to get this out of the PR queue

`//TODO check if token present`